hack/pkg/utils/longlivedsecrets.go

```diff
@@ -0,0 +1,362 @@
+package utils
```

Comment:
issue: Drop this file. The implementation in `test/helpers/longlivedsecret.go` is more or less identical.